pkg/controller/clusterstate/api/node_info.go

```diff
@@ -43,8 +43,10 @@ type NodeInfo struct {
 
```

Comment:
node_info.go is not used in preference for simplified cache inside the controller, @metalcycling can you please confirm the use of this file?